cocos/scene-graph/node.ts

```diff
@@ -1447,13 +1459,36 @@ export class Node extends CCObject implements ISchedulable, CustomSerializable {
      */
     public _static = false;
 
-    // world transform, don't access this directly
+    /**
+     * @engineInternal world transform, don't access this directly
+     */
+    public get w_pos () {
```

Comment:
The name `w_pos` doesn't match our coding style.

Reply:
updated https://github.com/cocos/cocos-engine/pull/14572/commits/467e70ef6c4d557169328ba54be7f90f0f59386e